app/main/controllers/upload/RTMediaUploadView.php

```diff
@@ -48,10 +48,18 @@ public function render( $template_name ) {
 			} elseif ( is_rtmedia_album_enable() && $rtmedia_query && is_rtmedia_gallery() ) {
 
 				if ( isset( $rtmedia_query->query['context'] ) && 'profile' === $rtmedia_query->query['context'] ) {
-					$album = '<span> <label> <i class="dashicons dashicons-format-gallery rtmicon"></i>' . esc_html__( 'Album', 'buddypress-media' ) . ': </label><select name="album" class="rtmedia-user-album-list">' . rtmedia_user_album_list() . '</select></span>';
+					$album_list_markup = rtmedia_user_album_list();
+
+					if ( ! empty( $album_list_markup ) ) {
+						$album = '<span> <label> <i class="dashicons dashicons-format-gallery rtmicon"></i>' . esc_html__( 'Album', 'buddypress-media' ) . ': </label><select name="album" class="rtmedia-user-album-list">' . $album_list_markup . '</select></span>';
```

Comment:
Is the output of `$album_list_markup` is escaped?

Reply:
Yes the function `rtmedia_user_album_list` and `rtmedia_group_album_list` functions returns html markup after escaping attributes and text contain.